### PR TITLE
added quickstart, datasource and dashboard for windows cert monitoring

### PR DESCRIPTION
**Pull request by Nimzr**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/windows/windows-certs/sec-icon.png
Comment (brammerl) [inline quickstarts/windows/windows-certs/sec-icon.png:1]: There can only be one logo per quickstart. Please choose either the logo.svg or this file as the logo to represent the quickstart. If you chose to go with this one, please rename to logo.png

File: data-sources/flex-install-windows-cert/config.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+id: flex-install-windows-cert
```
Comment (brammerl) [inline data-sources/flex-install-windows-cert/config.yml:1]: let's rename this `windows-certs` to create consistency with the how dashboards are referred to.

File: data-sources/flex-install-windows-cert/sec-icon.png
[the remark this answers is not in the document]
Comment (Nimzr) [reply data-sources/flex-install-windows-cert/sec-icon.png:1]: done 😄

File: quickstarts/windows/windows-certs/sec-icon.png
Comment (Nimzr) [reply quickstarts/windows/windows-certs/sec-icon.png:1]: should be good now

File: data-sources/flex-install-windows-cert/config.yml (hunk 1 above)
Comment (Nimzr) [reply data-sources/flex-install-windows-cert/config.yml]: this should be done now too, would love it if you can let me know if it looks good.

**Comment by seemantk** [comment]
hi @Nimzr thank you for making the requested changes, we'll review this again this week

File: data-sources/windows-certs/config.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  Using the infrastructure agent and a flex integration you can pull certification expiration using powershell and send the information to new relic to be used in dashboards or alerts.
+
+  Follow the regular flex installation process to add this flex integration to your agents, study the data to confirm if it matches your needs. 
+
+  ## What is Flex
+  New Relic Flex is an application-agnostic, all-in-one tool that allows you to collect metric data from a wide variety of services. It comes bundled with our infrastructure agent. You can instrument any app that exposes metrics over a standard protocol (HTTP, file, shell) in a standard format (for example, JSON or plain text): you create a YAML config file, start the Infrastructure agent, and your data is reported to New Relic.
+
+icon: logo.png   
+# Searchable words or phrases
+keywords:
```
Comment (josephgregoryii) [inline data-sources/windows-certs/config.yml:14]: Would we want to add in `microsoft` into our list of keywords?

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**